lib/moment/juttle-moment.js

```diff
@@ -844,6 +788,66 @@ var JuttleMoment = Base.extend({
         }
 
         return operand.valueOf();
+    },
+    momentString: function(moment) {
+        if (isNaN(moment.valueOf())) {
+            return moment._i.toString();
+        } else {
+            return moment.toDate().toISOString();
+        }
+    },
+    simpleDurationString: function(d) {
```

Comment:
around the time of shuttering, there was talk of removing this short form of duration string and only using the ASP.NET "TimeSpan" format (the thing that looks like a digital clock readout). People were often confused by this shorter form appearing in output.